deploy/helm/sumologic/upgrade-2.0.0.sh

```diff
@@ -181,17 +181,14 @@ function migrate_prometheus_recording_rules() {
   fi
 
   local RECORDING_RULES_OVERRIDE
-  RECORDING_RULES_OVERRIDE=$(yq r "${TEMP_FILE}" -- 'prometheus-operator.kubeTargetVersionOverride')
+  readonly RECORDING_RULES_OVERRIDE=$(yq r "${TEMP_FILE}" -- 'prometheus-operator.kubeTargetVersionOverride')
 
   if [[ "${RECORDING_RULES_OVERRIDE}" == "1.13.0-0" ]]; then
+    add_prometheus_pre_1_14_recording_rules "${TEMP_FILE}"
     info "Removing prometheus kubeTargetVersionOverride='1.13.0-0'"
     yq d -i "${TEMP_FILE}" "prometheus-operator.kubeTargetVersionOverride"
-    add_prometheus_pre_1_14_recording_rules "${TEMP_FILE}"
-  elif [[ -z "${RECORDING_RULES_OVERRIDE}" ]]; then
```

Comment:
what about this condition? What if `kubeTargetVersionOverride` is unset?

Reply:
We don't want to add pre 1.14 rules then.
As per discussion with @frankreno